src/components/SplitButton/__stories__/splitButton.stories.mdx

```diff
@@ -190,10 +252,11 @@ The split button has supports multiple sizes: small, medium and large.
         ),
         description: (
           <>
-            Don’t use split button if there’s only one option within the menu. Use
+            Don’t use split button if there’s only one option within the menu. Use{" "}
             <Link href="/?path=/docs/buttons-button--overview" withoutSpacing>
```

Comment:
maybe you can just remove without spacing

Reply:
oh 😓